src/targets/awsLambdaLayer.ts

```diff
@@ -171,23 +171,23 @@ export class AwsLambdaLayerTarget extends BaseTarget {
 
     await withTempDir(
       async directory => {
-        const git = simpleGit(directory);
         this.logger.info(
           `Cloning ${remote.getRemoteString()} to ${directory}...`
         );
-        await git.clone(remote.getRemoteStringWithAuth(), directory);
-
-        if (!isDryRun()) {
-          await this.publishRuntimes(
-            version,
-            directory,
-            awsRegions,
-            artifactBuffer
-          );
-          this.logger.debug('Finished publishing runtimes.');
-        } else {
-          this.logger.info('[dry-run] Not publishing new layers.');
-        }
+        const git = await cloneRepo(remote.getRemoteStringWithAuth(), directory);
+
+        await safeExec(
+          async () => {
+            await this.publishRuntimes(
```

Comment:
Explain what's going on here in comments please. Why are we running `this.publishRuntimes()` inside a `safeExec` callback?